File: src/main/java/com/example/camunda/delegate/FlowBDelegate.java
package com.example.camunda.delegate;

import org.camunda.bpm.engine.delegate.DelegateExecution;
import org.camunda.bpm.engine.delegate.JavaDelegate;
import org.springframework.stereotype.Component;

@Component("FlowBDelegate")
public class FlowBDelegate implements JavaDelegate {
    @Override
    public void execute(DelegateExecution execution) throws Exception {
        execution.setVariable("Hello","I am Flow B");
    }
}

File: backend/src/main/kotlin/com/github/jarryzhou/ptah/camunda/delegate/FlowBDelegate.kt
package com.github.jarryzhou.ptah.camunda.delegate

import org.camunda.bpm.engine.delegate.DelegateExecution
import org.camunda.bpm.engine.delegate.JavaDelegate
import org.springframework.stereotype.Component

@Component("FlowBDelegate")
class FlowBDelegate : JavaDelegate {
    @Throws(Exception::class)
    override fun execute(execution: DelegateExecution) {
        execution.setVariable("Hello", "I am Flow B")
    }
}
